### PR TITLE
Add the ability to change the DNS servers used by curl

### PR DESCRIPTION
**Pull request by dkurzaj**

### CODE REVIEW

File: src/robots.hpp [hunk 1]
```diff
@@ -113,6 +114,14 @@ class Robots
     return fetch_date;
   }
 
+  inline static void set_curl_dns_servers(std::string curl_dns_servers_p) {
```
Comment (NoelM) [inline src/robots.hpp]: A `const` argument would be great!

**Comment by NoelM** [comment]
I also requested Julien's review.

File: src/robots.hpp (hunk 1 above)
Comment (dkurzaj) [reply src/robots.hpp]: Resolved in: 421585c

File: src/network.cpp [hunk 2]
```diff
@@ -88,6 +91,11 @@ long curl_wraper(std::string& url,
     curl_easy_setopt(curl, CURLOPT_MAXREDIRS, 5L); // avoid infinite redirs by limiting to 5
     curl_easy_setopt(curl, CURLOPT_MAXFILESIZE, 1048576L); // set to 1MB max filesize
 
+    if (!curl_dns_servers.empty()) {
```
Comment (juldavid) [inline src/network.cpp]: This should be a static variable, setted by a static function.
Not a parameter of a function (lines 71 and 44)

Comment (dkurzaj) [reply src/network.cpp]: Does the changes made in 2879f85 resolve this?

File: src/network.cpp [hunk 3]
```diff
@@ -32,12 +33,23 @@
 #include <vector>
 #include <curl/curl.h>
 
+#include "robots.hpp"
```
Comment (juldavid) [inline src/network.cpp]: no need for this

**Pull request merged**